lib/StripeResource.js

```diff
@@ -134,13 +134,26 @@ StripeResource.prototype = {
       res.on('end', function() {
         var headers = res.headers || {};
 
+        // For convenience, make Request-Id easily accessible on
+        // lastResponse.
+        res.requestId = headers['request-id'];
+
+        var responseEvent = utils.removeEmpty({
+          api_version: req._requestEvent.api_version,
```

Comment:
Well, it's not actually a problem.  [Node's `http` module lowercases all header names](https://github.com/nodejs/node-v0.x-archive/issues/1954) because they're [case-insensitive](https://www.w3.org/Protocols/rfc2616/rfc2616-sec4.html) so:

1) It doesn't matter what Stripe sends because I should be accessing them up via their lowercase name in Node; and
2) I feel kinda bad that I didn't know this already.